### PR TITLE
Convert some more error tests

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by wthrowe — DISMISSED** [review]

File: tests/Unit/ApparentHorizons/Test_FastFlow.cpp [hunk 1]
```diff
@@ -307,34 +307,25 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.FastFlowMisc", "[Utilities][Unit]") {
   test_copy_and_move();
   test_serialize();
   test_ostream();
-}
```
Comment (knelli2) [inline tests/Unit/ApparentHorizons/Test_FastFlow.cpp:310]: Missing brace?

**Review by knelli2 — CHANGES_REQUESTED** [review]

**Comment by nilsdeppe** [comment]
Can you please run PRs through CI on your fork before opening a PR? The topic of CI time for PRs comes up fairly regularly and not having a bunch of PRs that fail several times would help that.

**Comment by nilsdeppe** [comment]
@knelli2 could you please look at this again?

File: src/DataStructures/SliceVariables.hpp [hunk 2]
```diff
@@ -96,11 +96,11 @@ void add_slice_to_data(
   const size_t volume_grid_points = extents.product();
   const size_t slice_grid_points = extents.slice_away(sliced_dim).product();
   ASSERT(volume_vars->number_of_grid_points() == volume_grid_points,
-         "volume_vars has wrong number of grid points.  Expected "
+         "volume_vars has wrong number of grid points. Expected "
```
Comment (knelli2) [inline src/DataStructures/SliceVariables.hpp]: Could you change this error message in a separate commit? It's unrelated to converting error tests

**Review by knelli2 — CHANGES_REQUESTED** [review]

File: src/DataStructures/SliceVariables.hpp (hunk 2 above)
Comment (nilsvu) [reply src/DataStructures/SliceVariables.hpp]: I just dropped the change

**Review by knelli2 — APPROVED** [review]

**Pull request merged**